<launch>
    <node pkg="touchosc_bridge" type="touchosc_bridge.py" name="touchosc" >
        <param name="osc_name" value="ROS OSC Default" />
        <param name="port" value="9000"/>
        <param name="print_fallback" value="True" />
        <param name="load_default" value="True" />
        <param name="publish_accel" value="True" />
        <param name="publish_diag" value="True" />
        <param name="vibrate" value="True" />
    </node>

    <node pkg="pytouchosc" type="layoutserver" name="layoutserver" output="screen">
        <param name="layout_path" value="$(find teleop_handler)/layouts" />
        <rosparam param="layouts">[ "teleop-ipod.touchosc", "teleop-ipad.touchosc" ]</rosparam> 
    </node>
</launch>
